src/docfx/lib/log/Report.cs

```diff
@@ -89,10 +104,11 @@ private static void ConsoleLog(ErrorLevel level, Error error)
             lock (Console.Out)
```

Comment:
>Console.Out [](start = 18, length = 11)

since output could be Console.Error  or Console.Out, should we lock output instead?